cmd/note_common.go

```diff
@@ -111,7 +226,12 @@ func createNote(rn string, isMR bool, idNum int, msgs []string, filename string,
 				"merged": "MERGED",
 			}[mr.State]
 
-			body = fmt.Sprintf("\n# This comment is being applied to %s Merge Request %d.", state, idNum)
+			if commit != "" {
```

Comment:
We need to check `len(commit)` here to for the minimum we are requiring, otherwise it'll panic in case the user pass something smaller than we expect. Or the check can be even earlier, when the `commit` flag is retrieved in `noteRunFn()`.

Reply:
Will do.  I will check it noteRunFn() to be consistent.

Reply:
Fixed in update.  Now errors out with:
```
[prarit@prarit rhel-8]$ ~/git-kernel/github/lab/lab mr note 649 --commit 8b37ff4a
2021/08/19 15:52:46 ERROR: note_common.go:76: Error: commit string length must be greater than 12 characters.
```